pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
+> Try also `ugrep --help {{something}}`.
+> More information: <https://github.com/Genivia/ugrep>.
+
+- Start a [q]uery TUI to search files in the working directory recursively (CTRL-Z for help):
+
+`ugrep -Q`
+
+- Search the working directory recursively for files containing a regex pattern:
+
+`ugrep "{{pattern}}"`
+
+- Search for a case-[i]nsensitive regex pattern:
+
+`ugrep -i "{{pattern}}"`
+
+- [L]ist the matching files:
+
+`ugrep -l "{{pattern}}"`
+
+- Search compressed files, [z]ip and tar archives recursively:
+
+`ugrep -z "{{pattern}}"`
+
+- Fu[z]zy search files with up to 3 extra, missing or mismatching characters in the pattern:
+
+`ugrep -Z3 "{{pattern}}"`
```

Comment:
```suggestion
`ugrep --fuzzy=3 "{{pattern}}"`
```

Reply:
Regrettably the Fu[z]zy sentence above it won't match the use of the `-Z` option any longer, right?

Reply:
We prefer long options when possible because they are easier to remember then short ones.